test/holder.go

```diff
@@ -44,11 +44,8 @@ func (h *Holder) Close() error {
 }
 
 // Reopen closes the holder and instantiates and opens a new holder.
+// note that the hold most be Closed first
```

Comment:
@travisturner Thanks I will adjust function comment to reflect the changed behavior.  Previously this method was swallowing errors on `Close()`.  To accomplish the goals of the initial test I thought it was better to run `Close()` as a preceding step, handling any error.  Then make file permission changes, that would later fail on `Reopen()`